cookbooks/bcpc-hadoop/recipes/namenode_standby.rb

```diff
@@ -84,6 +84,13 @@
   end
 end
 
+if node[:bcpc][:hadoop][:hdfs][:ldap][:integration] == true
+  template "/etc/hadoop/conf/ldap-conn-pass.txt" do
+    source "hdp_ldap-conn-pass.txt.erb"
+    mode 0655
+  end
+end
```

Comment:
We also likely want to set this to be a sensitive resource so the password is not echoed into the log.